app/models/equipment_model.rb

```diff
@@ -121,6 +126,12 @@ def not_associated_with_self
     attachment.instance.normalized_photo_name
   end
 
+  def assign_ordering
```

Comment:
this method definition belongs further down in the class -- it should be private